cypress/components/vertical-divider/vertical-divider.cy.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import CypressMountWithProviders from "../../support/component-helper/cypress-mount";
+import { assertCssValueIsApproximately } from "../../support/component-helper/common-steps";
+import { VerticalDividerComponent } from "../../../src/components/vertical-divider/vertical-divider-test.stories";
+
+import verticalDividerComponent from "../../locators/vertical-divider/index";
+import { VerticalDividerProps } from "../../../src/components/vertical-divider";
+
+context("Tests for VerticalDivider component", () => {
+  describe("should check VerticalDivider component properties", () => {
+    describe.each(["h", "height"])(
+      "when %s prop is passed as a string",
+      (propName) => {
+        it.each(["30px", "50px"])(
+          `verify that the ${propName} is set to %s for VerticalDivider component`,
+          (val) => {
+            const props = { [propName]: val };
+            CypressMountWithProviders(<VerticalDividerComponent {...props} />);
+
+            verticalDividerComponent()
+              .then(($el) => {
+                assertCssValueIsApproximately($el, "height", parseInt(val));
+              })
+              .should("have.attr", "height", val);
+
+            cy.checkAccessibility();
```

Comment:
Do you think it's ok to leave the accessibilty and functional testing in the same test?